yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -163,24 +163,38 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 		return len(constructs)
 	})
 
-	assessPayload := callhome.AssessMigrationPhasePayload{
-		TargetDBVersion:     assessmentReport.TargetDBVersion,
-		MigrationComplexity: assessmentReport.MigrationComplexity,
-		UnsupportedFeatures: callhome.MarshalledJsonString(lo.Map(assessmentReport.UnsupportedFeatures, func(feature UnsupportedFeature, _ int) callhome.UnsupportedFeature {
+	var unsupportedFeatures []callhome.UnsupportedFeature
+	for _, feature := range assessmentReport.UnsupportedFeatures {
+		if feature.FeatureName == EXTENSION_FEATURE {
+			// For extensions, we need to send the extension name in the feature name
+			// for better categorization in callhome
+			for _, object := range feature.Objects {
+				unsupportedFeatures = append(unsupportedFeatures, callhome.UnsupportedFeature{
+					FeatureName:      fmt.Sprintf("%s - %s", feature.FeatureName, object.ObjectName),
+					ObjectCount:      1,
+					TotalOccurrences: 1,
+				})
+			}
+		} else {
 			var objects []string
 			if slices.Contains(featuresToSendObjectsToCallhome, feature.FeatureName) {
 				objects = lo.Map(feature.Objects, func(o ObjectInfo, _ int) string {
 					return o.ObjectName
 				})
 			}
```

Comment:
maybe we can get rid of it as it was done only for extension earlier?

Reply:
I think it might be useful to leave this logic in. We did want to do this for a few issue types (such as xml functions). 

I'll remove the extensions issue from the `featuresToSendObjectsToCallhome` list though.